docs/source/python_api/mlflow.metrics.rst

```diff
@@ -146,20 +146,20 @@ Parameters:
             model_type="retriever",
             extra_metrics = [
                 mlflow.metrics.precision_at_k(5),
-                mlflow.metrics.precision_at_k(6),
-                mlflow.metrics.recall_at_k(4),
-                mlflow.metrics.recall_at_k(5)
+                mlflow.metrics.recall_at_k(5),
+                mlflow.metrics.ndcg_at_k(5)
             ]   
         )
     
     NOTE: In the 2nd method, it is recommended to omit the ``model_type`` as well, or else 
     ``precision@3`` and ``recall@3`` will be  calculated in  addition to ``precision@5``, 
-    ``precision@6``, ``recall@4``, and ``recall@5``.
+    ``recall@5``, and ``ndcg_at_k@5``.
```

Comment:
ndcg@3 will be calculated by default as well

maybe to match the previous format, you can write ndcg@5?

I also originally wanted this to show how you can use diff values of k, so maybe also calculate precision@6 along with these 3 @5 metrics

Reply:
Got it, sgtm!